sound/soc/sof/sof-audio.h

```diff
@@ -637,6 +637,7 @@ int get_token_comp_format(void *elem, void *object, u32 offset);
 int get_token_dai_type(void *elem, void *object, u32 offset);
 int get_token_uuid(void *elem, void *object, u32 offset);
 int get_token_string(void *elem, void *object, u32 offset);
+int get_token_void(void *elem, void *object, u32 offset);
```

Comment:
@jsarha this is unnecessary. You dont really need to parse the token anywwhere at all. Instead what you can do in in sof_widget_ready() in topology.c, iterate though the list of tuples in the swidget's tuples-array, find the token you are insterested in and set the field in swidget based on that

Reply:
just to be clearer, this token is IPC agnostic. So why restrict it to IPC4? it should go to the common topology parser